Monika After Story/game/script-greetings.rpy

```diff
@@ -1571,6 +1571,24 @@ label monikaroom_greeting_ear_renpy_docs:
 
     jump monikaroom_greeting_choice
 
+# Monika is singing a lovely song while you're away
+init 5 python:
+    if persistent._mas_affection.get("affection", 0) >= 100:
+        gmr.eardoor.append("monikaroom_greeting_ear_somebody_to_love")
```

Comment:
with affection, you should add the gmr.eardoor append to the affection progpoints, and just not have an init 5 for this label.